test/basic_tests.py

```diff
@@ -144,70 +149,80 @@ def test_run_tf_model(env):
     env.assertEqual(type(exception), redis.exceptions.ResponseError)
 
     try:
-        env.execute_command('AI.MODELSET', 'm_1', 'TF',
-                            'INPUTS', 'a', 'b', 'OUTPUTS', 'mul', model_pb)
+        if TEST_TF:
```

Comment:
Wouldn't this (and all the similar cases below) fail if `TEST_TF` is `False` since `exception` will not be raised here?

Reply:
But of course. The whole test is TF so it can be controlled from the head.